dids/src/main/kotlin/web5/sdk/dids/methods/jwk/DidJwk.kt

```diff
@@ -144,36 +144,33 @@ public class DidJwk(uri: String, keyManager: KeyManager) : Did(uri, keyManager)
         throw IllegalArgumentException("decoded jwk value cannot be a private key")
       }
 
-      val verificationMethodId = URI.create("$did#0")
-      val verificationMethod = VerificationMethod.builder()
+      val verificationMethodId = "${parsedDidUri.uri}#0"
+      val verificationMethod = VerificationMethod.Builder()
         .id(verificationMethodId)
-        .publicKeyJwk(publicKeyJwk.toJSONObject())
-        .controller(URI(did))
-        .type("JsonWebKey2020")
+        .publicKeyJwk(publicKeyJwk)
+        .controller(did)
+        .type("JsonWebKey")
         .build()
 
-      val verificationMethodRef = VerificationMethod.builder()
-        .id(verificationMethodId)
-        .build()
-
-      val didDocumentBuilder = DIDDocument.builder()
-        .contexts(
-          mutableListOf(
-            URI.create("https://w3id.org/security/suites/jws-2020/v1")
-          )
-        )
-        .id(URI(did))
-        .verificationMethod(verificationMethod)
+      val didDocumentBuilder = DIDDocument.Builder()
+        .context("https://www.w3.org/ns/did/v1")
```

Comment:
since `"https://www.w3.org/ns/did/v1"` [is always to be included](https://www.w3.org/TR/did-core/#production-0) in a DID Document we could encapsulate that within the `DIDDocument` abstraction

Reply:
the spec is confusing here, but this is not the case. the section you pointed to is for JSON-LD; however, there is [a section above](https://www.w3.org/TR/did-core/#json) on a pure JSON representation that does not have this restriction.

for DID DHT I've chosen to follow the JSON representation and explicitly excluded the context property

Reply:
nice!